config/onebusaway-transit-data-federation-webapp-data-sources.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xmlns:context="http://www.springframework.org/schema/context"
+xsi:schemaLocation="
+    http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
+    http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-3.0.xsd">
+
+    <bean id="dataSource" class="org.springframework.jdbc.datasource.DriverManagerDataSource">
+        <property name="driverClassName" value="org.hsqldb.jdbcDriver" />
+        <property name="url" value="jdbc:hsqldb:file:/bundle/org_onebusaway_transit_data" />
+        <property name="username" value="sa" />
+        <property name="password" value="" />
+    </bean>
+    <bean class="org.onebusaway.container.spring.SystemPropertyOverrideConfigurer">
+      <property name="order" value="-2" />
+      <property name="properties">
+          <props>
+              <prop key="bundlePath">/var/lib/obanyc/no-such-dir</prop>
+         </props>
+      </property>
+    </bean>
+    <bean id="httpServiceClient" class="org.onebusaway.transit_data_federation.util.HttpServiceClientImpl" >
```

Comment:
Maybe this section isn't needed until later but does it seem sane?

Reply:
I believe `httpServiceClient` should be in `data-sources.xml` for `onebusaway-enterprise-acta-webapp`, not `transit-data-fed-webapp`.

Reply:
Did try removing this and it broke, exception [here](https://gist.github.com/danp/607fcb354e4ea91b77b116f29b47043a). Will leave in for now.